webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin")
+
+const HtmlWebpackPluginConfig = new HtmlWebpackPlugin({
+    template: './public/index.html',
+    filename: 'index.html',
+    inject: 'body'
+})
+
+module.exports = {
+    entry: "./public/index.js",
+    output: {
+        path: path.join(__dirname, 'build'),
+        filename: 'index.bundle.js'
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
```

Comment:
Add  **JSX** file extension support as well because We'll be giving **React Components** the **JSX** extension.

Reply:
Sure, It's Added!